middleware/security.py

```diff
@@ -80,25 +82,27 @@ def decorator(*args, **kwargs):
             if len(authorization_header) >= 2 and authorization_header[0] == "Bearer":
                 api_key = request.headers["Authorization"].split(" ")[1]
                 if api_key == "undefined":
-                    return {"message": "Please provide an API key"}, 400
+                    abort(code=400, message="Please provide an API key")
```

Comment:
@maxachis Is this going to change the shape of the error objects we receive on the client?

Reply:
@joshuagraber Based on my tests, yes, but only somewhat, and possibly not in a way that interferes. The `response.json` will now include a "http_status_code" key-value pair in addition to the "message" key-value pair. 

So `{'message': 'An unexpected error occurred'}` will become `{'http_status_code': 500, 'message': 'An unexpected error occurred'}`

Reply:
No problem @maxachis thank you.